plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMergeTableHandle.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.trino.spi.connector.ConnectorMergeTableHandle;
+
+import java.util.Optional;
+
+public class JdbcMergeTableHandle
+        implements ConnectorMergeTableHandle
+{
+    private final JdbcTableHandle tableHandle;
+    private final JdbcOutputTableHandle outputTableHandle;
+    private final Optional<JdbcOutputTableHandle> deleteTableHandle;
```

Comment:
This field is only used when you add support for merge to postgres, so I believe this is when this field should be added.